metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -478,6 +481,33 @@ case class ScalafixProvider(
       }
   }
 
+  private def askForRule(rules: Set[String]): Future[Option[String]] =
+    languageClient
+      .metalsQuickPick(
+        MetalsQuickPickParams(
```

Comment:
Would it be necessary to check the client is a quick pick provider?

Reply:
We do check that inside ConfiguredLanguageClient